engine/src/main/java/org/camunda/bpm/engine/impl/db/EnginePersistenceLogger.java

```diff
@@ -792,4 +792,9 @@ public void installationIdPropertyFound(String value) {
         "100", "Installation id property found in the database: {}", value);
   }
 
+  public void ignoreFailureDuePreconditionNotMet() {
+    logDebug(
+        "101", "Ignoring operation failure due to unmet precondition");
```

Comment:
Is it possible to pass some contextual info here? E.g. "parent operation failed".

Reply:
Also which operation this message concerns.

Reply:
How does it sound:
```Ignoring database operation failure due to an unmet precondition. Parent database operation failed: 'UPDATE EverLivingJobEntity[1]'```

Reply:
Sounds good. If possible, include_ "DELETE ByteArray[X]" as well so the message is self-contained.